docs/guis/multiturn/acdipole.md

```diff
@@ -0,0 +1,76 @@
+# AC-Dipole Measurements
+Once all the steps have been followed in the [Multiturn GUI](gui.md), you can start the measurements with the AC-dipole.
+
+See the gui below for the `ACDipole` tab in the Multiturn GUI:
+<figure>
+    <center>
+    <img src="../../assets/images/multiturn_gui/default_view.png" width="85%" alt="AC-Dipole Tab" />
+    <figcaption>AC-Dipole Tab</figcaption>
+    </center>
+</figure>
+
+There are two important steps to take before starting the measurements:
+1. Set the Tune Deltas, this is done by changing the `start` text fields in the `Tune deltas` of the `Horizontal settings` and `Vertical settings` sections of the GUI:
+   - The horizontal tune delta is typically set to **-0.01**.
+   - The vertical tune delta is typically set to **0.012**.
+   - These values result in typical excitation tunes of **Qx = 0.27** and **Qy = 0.322**.
```

Comment:
I think this would better be an admonition saying it's a bit of a default for most measurements but it's not necessarily the case (there are ecloud tunes, tunes for specific RDTs, nominal tunes at end of squeeze etc)